Foundation/NSDate.swift

```diff
@@ -165,6 +165,15 @@ open class NSDate : NSObject, NSCopying, NSSecureCoding, NSCoding {
     }
 }
 
+// SPI for XCTest
+#if os(Windows)
+extension NSDate {
+  public static func _currentAbsoluteTime() -> TimeInterval {
+    return CFAbsoluteTimeGetCurrent()
```

Comment:
Isnt this just `Date.timeIntervalSinceReferenceDate` ? If so we could just fixup XCTest to use that directly